src/routes/(app)/app/[workspaceSlug]/integrations/+page.svelte

```
@@ -301,6 +342,45 @@
 		return ALL_PROVIDERS.find((provider) => provider.key === providerKey)?.name ?? providerKey;
 	}
 
+	async function clearMarketingCallbackParams() {
+		const params = new URLSearchParams(page.url.searchParams);
```

ESLint

[error] Found a mutable instance of the built-in URLSearchParams class. Use SvelteURLSearchParams instead.

(svelte/prefer-svelte-reactivity)